lint/rule_urls.go

```diff
@@ -8,13 +8,14 @@ import "strings"
 // At Google, we inject additional rule URL mappings into this list.
 // Example: google_rule_url_mappings.go
 // package lint
-// func init() {
-//   ruleURLMappings = append(ruleURLMappings, internalRuleURLMapping)
-// }
 //
```

Comment:
Is this change intentional?

Reply:
I reverted the change, it's due to go 1.19's `gofmt`